metrics/web/lib/common/domain/usecases/parameters/instant_config_param.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:equatable/equatable.dart';
+import 'package:meta/meta.dart';
+
+/// Represents an instant config parameter.
+class InstantConfigParam extends Equatable {
+  /// Indicates whether to show the login form.
+  final bool isLoginFormEnabled;
+
+  /// Indicates whether the FPS monitor feature is enabled.
+  final bool isFpsMonitorEnabled;
+
+  /// Indicates whether the renderer display feature is enabled.
+  final bool isRendererDisplayEnabled;
+
+  @override
+  List<Object> get props => [
+        isLoginFormEnabled,
+        isFpsMonitorEnabled,
+        isRendererDisplayEnabled,
+      ];
+
+  /// Creates a new instance of the [InstantConfigParam]
+  /// with the given [isLoginFormEnabled], [isFpsMonitorEnabled],
+  /// and [isRendererDisplayEnabled].
```

Comment:
```suggestion
  /// Creates a new instance of the [InstantConfigParam]
  /// with the given config parameters.
```
The list of parameters can change and we'll have to change this documentation of the constructor so I would suggest avoiding exposing parameters in such a way.